lib/fathead/perl_doc/Parsers/parse-faq.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
```

Comment:
Just to point out also, this was just my idea of a flexible approach. It may turn out that throwing lists of pages at ad-hoc scripts ends up being simpler and more maintainable. I think the DBI thing is probably the most valuable part of the script - manually counting tab characters doesn't feel robust to me.

Going to leave it in your hands - if you want to keep going with parse-faq, that's absolutely cool :smile: